src/components/Wizard/CreateImageWizard.js

```diff
@@ -142,6 +144,17 @@ const CreateImageWizard = (props) => {
             tenancy: formValues.image.upload.settings.tenancy,
           },
         };
+      } else if (formValues?.image?.type === "gce") {
+        uploadSettings = {
+          image_name: formValues.image.upload.image_name,
+          provider: "gcp",
+          settings: {
+            region: formValues.image.upload.settings.region,
+            bucket: formValues.image.upload.settings.bucket,
+            object: formValues.image.upload.settings.object,
```

Comment:
I should update the osbuild guides, but the `object` is no longer required to be provided. It can be, but IMO it does not make much sense. The reason is that the object name is used only temporarily for the image when uploaded to GCP Storage for importing and the object is afterwards deleted. Composer will generate a unique object name if it was not provided.

The relevant change is https://github.com/osbuild/osbuild-composer/pull/3023

Reply:
Ok. Ya I removed this.